bedrock/base/templates/macros.html

```diff
@@ -452,3 +450,304 @@
+        <li><a rel="external" href="https://support.mozilla.org/kb/private-browsing-browse-web-without-saving-info" data-ga="desktop - private browsing">{{ _('Private Browsing') }}</a></li>
+        <li><a rel="external" href="https://addons.mozilla.org" data-ga="desktop - addons">{{ _('Add-ons') }}</a></li>
+        <li><a rel="external" href="https://support.mozilla.org/products/firefox" data-ga="desktop - support">{{ _('Need help?') }}</a></li>
+      </ul>
+
+      <ul class="tertiarynav" data-parent="android">
+        <li><a href="{{ url('firefox.dnt') }}" data-ga="android - dnt">{{ _('Do Not Track') }}</a></li>
+        <li><a rel="external" href="https://www.youtube.com/watch?v=Ewk0YtFTBec&amp;list=PLA8ABF6B7A66A0C5E&amp;index=15" data-ga="android - private browsing">{{ _('Private Browsing') }}</a></li>
+        <li><a rel="external" href="https://addons.mozilla.org/android/" data-ga="android - addons">{{ _('Add-ons') }}</a></li>
+        <li><a rel="external" href="https://support.mozilla.org/products/mobile" data-ga="android - support">{{ _('Need help?') }}</a></li>
+      </ul>
+
+      <ul class="tertiarynav" data-parent="os">
+        <li><a rel="external" href="https://blog.mozilla.org/blog/category/firefox-os/" data-ga="os - news">{{ _('News') }}</a></li>
+        <li><a rel="external" href="https://developer.mozilla.org/Firefox_OS" data-ga="os - developers">{{ _('For developers') }}</a></li>
+        <li><a rel="external" href="https://find.firefox.com" data-ga="os - find my device">{{ _('Find My Device') }}</a></li>
+        <li><a rel="external" href="https://support.mozilla.org/products/firefox-os" data-ga="os - support">{{ _('Need help?') }}</a></li>
+      </ul>
+    </div>{#-- /#fxfamilynav-tertiarynav --#}
+
+    {# Page will dictate CTA (button, usually) to be placed in this wrapper #}
+    <div id="fxfamilynav-cta-wrapper" class="fxfamilynav-cta-wrapper"></div>
+
+    <button type="button" id="fxfamilynav-tertiarynav-trigger" class="fxfamilynav-tertiarynav-trigger" aria-controls="fxfamilynav-tertiarynav">
+      <span>{{ _('Menu') }}</span>
+      <div class="trigger-dots">
+        <div class="trigger-dot"></div>
+        <div class="trigger-dot"></div>
+        <div class="trigger-dot"></div>
+      </div>
+    </button>
+
+    <div id="fxfamilynav-tabzilla" class="fxfamilynav-tabzilla"><a href="{{ url('mozorg.home') }}">{{ _('Mozilla') }}</a></div>
+  </div>{#-- /.container --#}
+</header>{#-- /#fxfamilynav-header --#}
+
+<header id="fxfamilynavV2-header" class="fxfamilynav-header{% if (navclass) %} {{ navclass }}{% endif %}">
+  <div class="container">
+    <i class="fxfamilynav-logo">{{ _('Firefox') }}</i>
+    <nav id="fxfamilynavV2" class="fxfamilynav" role="navigation">
+      <ul id="fxfamilynavV2-primary" class="primarynav">
+        <li id="desktops">
+          <a href="#" class="primary-link" data-id="desktops"><div>{{ _('For desktops &amp; laptops') }}</div></a>
```

Comment:
Ah ok, so I should ignore if I resize the browser window and only look at what is there on page load, right? Just making sure I understand the logic.